apps/sv/frontend/src/components/ValidatorOnboardingSecrets.tsx

```diff
@@ -5,28 +5,47 @@ import {
   DisableConditionally,
   Loading,
   SvClientProvider,
-  CopyableTypography,
 } from '@lfdecentralizedtrust/splice-common-frontend';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
 import { useMutation } from '@tanstack/react-query';
-import React from 'react';
+import React, { useState, useCallback } from 'react';
 
-import { Button, Stack, Table, TableContainer, TableHead, Typography } from '@mui/material';
+import {
+  Button,
+  IconButton,
+  Stack,
+  Table,
+  TableContainer,
+  TableHead,
+  TextField,
+  Typography,
+} from '@mui/material';
 import TableBody from '@mui/material/TableBody';
 import TableCell from '@mui/material/TableCell';
 import TableRow from '@mui/material/TableRow';
 
 import { useSvAdminClient } from '../contexts/SvAdminServiceContext';
 import { useValidatorOnboardings } from '../hooks/useValidatorOnboardings';
 import { useSvConfig } from '../utils';
+import { useNetworkInstanceName } from '../hooks';
+import dayjs from 'dayjs';
+import {
+  dateTimeFormatISO,
+  getUTCWithOffset,
+} from '@lfdecentralizedtrust/splice-common-frontend-utils';
+
+const VALID_PARTY_ID_REGEX = /^[^-]+-[^-]+-\d+$/;
```

Comment:
Let's put this in a util file in the common package. I have a feeling we'll be reusing this again.

Reply:
In the past, i've seen regexes, numbers(e.g duration) used in sv and scan for instance but duplicated. Happy to leave for now

Reply:
yeah that's not what code reuse is about IMHO unless you have a clear higher level plan for modules / libs where to put these